packages/core/src/ControlHub.ts

```diff
@@ -1,4 +1,18 @@
-import { ExpansionHub } from "./ExpansionHub.js";
+import { ExpansionHub, ParentExpansionHub } from "./ExpansionHub.js";
 import { ParentRevHub } from "./RevHub.js";
+import { ModuleStatus } from "./ModuleStatus.js";
 
-export interface ControlHub extends ExpansionHub, ParentRevHub {}
+export interface ControlHub extends ExpansionHub, ParentRevHub {
+    on(eventName: "error", listener: (error: Error) => void): this;
+    on(eventName: "statusChanged", listener: (status: ModuleStatus) => void): this;
```

Comment:
Do we want to add these to ExpansionHub? This would improve the drop-in property of the two classes, but add complexity to the ExpansionHub class.